FreeRTOS/Test/CMock/queue/generic/queue_create_dynamic_utest.c

```diff
@@ -38,6 +38,7 @@
 #include "queue.h"
 
 /* ============================  GLOBAL VARIABLES =========================== */
+extern bool xMaskAssertAndAbort;
```

Comment:
I'm not sure I understand why this needs to be moved around.

Reply:
Get/Set functions would be cleaner if declaring this in the header file is problematic.

Reply:
we can't declare variables inside header files with multiple translation units
the linker will be confused with multiple declarations.
Yeah +1 for Set/Get functions